packages/rspack/tests/statsCases/issuse-3558/module/jsonpath.js

```diff
@@ -0,0 +1,6842 @@
+/*! jsonpath 1.1.1 */
```

Comment:
Would you move this to the fixtures?

Reply:
ok

Reply:
And hand write the minimal issue code instead of copy all jsonpath.js to keep the noise as low as possible

Reply:
> And hand write the minimal issue code instead of copy all jsonpath.js to keep the noise as low as possible

Agree

Reply:
We shouldn't need this file, just verify that we can resolve the redefined require